contracts/margin/external/ERC20/ERC20Position.sol

```diff
@@ -330,23 +322,29 @@ contract ERC20Position is
      * yourself. Likely, rounding error will be small enough to not properly incentivize people to
      * carry out such an attack.
      *
-     * @param  who  Address of the account to withdraw for
-     * @return      The amount of heldToken withdrawn
+     * @param  who  Array of addresses to withdraw for
+     * @return      The amount of heldToken withdrawn for each address
      */
-    function withdraw(
-        address who
+    function withdrawMultiple(
```

Comment:
As per our discussion, you either need to:

- Leave this as it was before
- Change all of these types of functions to only have one version that takes an array, and name it `withdraw`

Either way, this should be done in a separate PR